storybook-example/version_build.js

```diff
@@ -22,31 +13,57 @@ const printMessage = (message) => {
   console.log("\n");
 };
 
+// method for installing node modules
+const installNodeModule = () => {
+  const path = require("path").resolve("./", profile);
+  const dotenv = require("dotenv").config({
+    path,
+  }).parsed;
+
+  const pluginsFolderList =
+    dotenv.PLUGINS_FOLDER.trim() === ""
+      ? []
+      : dotenv.PLUGINS_FOLDER.split(",").map((_) => _.trim());
+
+  pluginsFolderList.forEach((_) => {
+    printMessage(`Changing directory to ../${_} and installing node modules`);
+    execSync(`cd ../${_} && (npm ci || npm i)`);
+  });
+};
+
+// install node module for the current branch
+installNodeModule();
+
 // executing the parent storybook which contain about page
 printMessage("Start building storybook-example");
 execSync("npm run build");
 printMessage("Storybook-example building complete");
 
-versionBranch.forEach((currentBranch) => {
-  // add script to checkout the current branch
-  scriptList.push({
-    cmd: `git checkout ${currentBranch}`,
-    msg: `Checkout to branch ${currentBranch}`,
-  });
+// default storybook build command
+const storybookBuildCmd = `set PROFILE=${profile} && npm run build -- -o storybook-static`;
 
-  // building storybook for current branch
-  scriptList.push({
-    cmd: `${storybookBuildCmd}/${currentBranch}`,
-    msg: `Building storybook for ${currentBranch} branch`,
-  });
-});
+// using the branch name from this list, to create version
+const versionBranch =
+  dotenv.VERSION_BRANCH.trim() === ""
+    ? []
+    : dotenv.VERSION_BRANCH.split(",").map((_) => _.trim());
+
+versionBranch.forEach((branch) => {
+  // execute script to checkout to the new branch
+  printMessage(`Checkout to branch ${branch}`);
+  execSync(`git checkout ${branch}`);
+
+  // installing node modules for plugins folder
+  // in checkout branch
+  installNodeModule();
 
-// executing script serially
-scriptList.forEach((_) => {
-  printMessage(_.msg);
-  execSync(_.cmd);
+  // building storybook for checkout branch
+  printMessage(`Building storybook for ${branch} branch`);
+  execSync(`${storybookBuildCmd}/${branch}`);
 });
 
 // Deploying storybook in github pages
-printMessage("Deploying storybook in github pages");
-execSync("npm run deploy");
+if (profile === ".env.production") {
```

Comment:
deployment script can be removed from here

Reply:
remove in this [0d71812](https://github.com/mosip/mosip-plugins/pull/58/commits/0d71812a3dfb6f5b38806a36638a1e7bc0401412)